code/modules/chemistry/chemicompiler_core.dm

```diff
@@ -393,7 +394,7 @@
 	if(source < minReservoir || source > maxReservoir)
 		throwError(CC_ERROR_INVALID_SX) // Invalid source id.
 		return
-	if(target < minReservoir || target > maxReservoir + 3)
+	if(target < minReservoir || target > maxReservoir && !target == utilReservoir)
```

Comment:
`utilReservoir` is being defined as a list though - I don't see how this code would actually work?

Reply:
thank you and fixed